cmd/world/forge/common.go

```
@@ -0,0 +1,148 @@
+			return false
+		}
+	}
+	return true
+}
+
+var generateKey = func() string {
+	return strings.ReplaceAll(uuid.NewString(), "-", "")
+}
+
+// Change from function to variable
+var openBrowser = func(url string) error {
```

golangci-lint (1.62.2)

var must not be placed after func (desired order: const,var,type,func)

(decorder)